site/multiDim/MultiDimDataPage.tsx

```diff
@@ -31,19 +31,24 @@ export function MultiDimDataPage({
         relatedResearchCandidates,
         imageMetadata,
         tagToSlugMap,
-        initialQueryStr,
     }
+    // Due to thumbnails not taking into account URL parameters, they are often inaccurate on
+    // social media. We decided to remove them and use a single thumbnail for all charts.
+    // See https://github.com/owid/owid-grapher/issues/1086
```

Comment:
Huh? We _do_ show chart previews based on query params

See [`rewriteMetaTags`](https://github.com/owid/owid-grapher/blob/master/functions/_common/grapherTools.ts#L196) 

<img width="1209" alt="image" src="https://github.com/user-attachments/assets/60f518db-b330-4fa0-b947-0e23239c904d" />

<img width="1161" alt="image" src="https://github.com/user-attachments/assets/3d7309c4-f4be-412c-ba3b-5b5125a11ad1" />

So at least one of us must be confused 😅

Reply:
Once we've resolved this, I'll actually test the PR 👍

Reply:
Oh, that's pretty cool that we do that. I copy-pasted the existing code from the data page, so I'll remove those comments in both places then. And, it works the same for mdims out of the box:

http://staging-site-mdim-thumbnails/grapher/mdd-demo-poverty?povertyLine=30&metric=share

Reply:
Testing mdims locally is quite complicated. You need a relatively fresh copy of the DB. The previews are available at and admin sub-path, e.g. `http://localhost:3030/admin/grapher/mdd-demo-poverty` (even if the mdim is not published). Let me know if you'll need help with it.